projects/Apps/common/src/index.ts

```diff
@@ -218,15 +218,7 @@ export const sizeDescriptions: { [k in ImageSize]: number } = {
     tabletXL: 1140,
 }
 
-export const Editions = [
```

Comment:
These changes are already in other related PR, do you want to merge this PR on top of it instead of master?